roles/openshift_web_console/vars/openshift-enterprise.yml

```diff
@@ -0,0 +1,4 @@
+---
+__openshift_web_console_prefix: "registry.access.redhat.com/openshift3/"
+__openshift_web_console_version: "v3.7"
```

Comment:
`v3.9` most likely what you're after here.

Across the project we're hardcoding versions in this manner, once we've cleaned up how we set the version and release we'll come back through and update everything to simply match the installed product vX.Y.